frontend-vue/src/routes.ts

```diff
@@ -41,10 +42,15 @@ export default [
     path: "/boundedContext/:id/namespaces",
     component: BoundedContextNamespaces,
   },
+  {
+    name: routes.Analytics,
+    path: "/analytics",
+    component: Analytics,
+  },
   {
     name: routes.Search,
     path: "/search",
-    component: Search,
+    component: Root,
```

Comment:
Here I would introduce a new component instead of removing it.

Reply:
Good point, added its own component